_posts/2020-09-15-newpipe-0.20.0.md

```diff
@@ -0,0 +1,114 @@
+
+It's 2020, and our default resolutions also realized this. Instead of having 360p everywhere, main player videos are opened in 720p at 60FPS now, and pop-up ones in 480p. Thank you, [@B0pol](https://github.com/B0pol). NewPipe now also remembers what kind of media you downloaded the last time, and will auto-select it when you hit download on a new video. This was done by [@vmazoyer](https://github.com/vmazoyer).
+
+Since 'Autoplay' is a new configurable setting in this release, [@opusforlife2](https://github.com/opusforlife2) changed the 'Autoplay' toggle in the video details to say "Auto-queue" instead. Now it's clearer what that toggle does without leaving room for any confusion.
+
+## Fixes
+
+If you noticed that next to offline search input now working, the actual text of the suggestions look better now, you are correct. [@TobiGr](https://github.com/TobiGr) changed the colours used to the default ones.
+
+An empty error report in certain kinds of crashes is not empty any more, since [@Stypox](https://github.com/Stypox) took care of properly calling code. Yay, code!
+
+Thanks to [@wb9688](https://github.com/wb9688), you can open a video in the local playlist tab, whereas earlier, the app would crash. And thanks to [@BoFFire](https://github.com/BoFFire), if you open the app in the Taqbaylit language, its called `Taqbaylit` now and not `Tamaziɣt Taqbaylit`.
+
+The license pop-up doesn't disappear when you change orientation now. This was introduced by [@nmurali94](https://github.com/nmurali94). And [@budde92](https://github.com/budde25) ensured that you can now delete a file you just downloaded and then try to re-download it without the app crashing. We are not sure why you'd want to do this, but now you can do so without any worries.
+
+[@mhmdanas](https://github.com/mhmdanas) changed a code call because its origin was redundant. Yay less redundancy. And [@wb9688](https://github.com/wb9688) fixed a crash when a video wouldn't have any next videos but NewPipe wanted to add one to the queue since auto-play were enabled.
+
+
+## Development
+
+Development. Where all the magic happens which keeps this app running. It saw some improvements in this release (Warning: Nerd talk ahead): A checkstyle rule was added so it would error when a local variable which could be `final`wasn't declared as such (by [@XiangRongLin](https://github.com/XiangRongLin)), numbers and letters are allowed in the app package id (by [@Stypox](https://github.com/Stypox)), contributors need to to discuss their contributions now prior to open a PR (by [@gkeegan](https://github.com/gkeegan)) and some depreciations as well as a little code cleanup happened (by [@TacoTheDank](https://github.com/TacoTheDank).
```

Comment:
It's "Checkstyle" and you're missing a space after `final`. It's "application ID", not "app package id", and it's "deprecations". Please end your file with a newline.